circle-test.sh

```diff
@@ -9,18 +9,12 @@ set -e
 DIR=$(cd $(dirname "${BASH_SOURCE[0]}") && pwd)
 cd $DIR
 
-export OUTPUT_DIR="$CIRCLE_ARTIFACTS" 
+export OUTPUT_DIR="$CIRCLE_ARTIFACTS"
```

Comment:
Oh.. this line looks like it had trailing white space committed for some reason. My editor removed it.